src/phantom/schema.py

```diff
@@ -22,7 +25,25 @@ class Schema(TypedDict, total=False):
     maxLength: Optional[int]
 
 
+def desc_from_docstring(cls) -> str:
+    # ds = (next(filter(lambda x: x.__doc__, cls.__mro__), cls).__doc__ or "").strip()
```

Comment:
```suggestion
```

issue: Don't leave commented out code.

Reply:
Concerning your proposal, I simply did not see how intended behavior with inheritance was supposed to be and opted for it. I will remove it.

At first I found your example D unintuitive, but if the intent is to pass use_docstring on each subclass, I agree it makes sense. Will adapt it accordingly.

Reply:
Yes, I would have removed it. Was not sure about which version to use depending on desired behavior. But as we don't do inheritance for the flag, I can remove it.